x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/support/StringMatcherTests.java

```diff
@@ -129,19 +129,32 @@ public void testMultiplePatterns() throws Exception {
             List.of(prefix1 + "*", prefix2 + "?", "/" + prefix3 + "@/", "*" + suffix1, "/@" + suffix2 + "/", exact1, exact2, exact3)
         );
 
+        final List<Character> nonAlpha = "@/#$0123456789()[]{}<>;:%&".chars()
+            .mapToObj(c -> Character.valueOf((char) c))
```

Comment:
Optional nit: I think boxing happens automatically here, i.e., `c -> (char) c)` is sufficient.

Reply:
It would, though I think explicit is preferable in this case.
I guess it's just going to come down to personal preference, but I like the clarity of the explicit boxing.